lhcb-YYYY-open-data-curation/scripts/sanitizeStripping.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
Cosmetics: we could pass the shell scripts via `shellcheck` , there are several observations to improve some quote expansions etc.

Reply:
Sure, I will look into that.